### ISSUE TITLE
Claw hand in Charcot Marie Tooth

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7670]
Our text mining for CMT is picking up claw hand as [Split hand](https://hpo.jax.org/app/browse/term/HP:0001171) but this is not what is meant, it is https://charcot-marie-toothnews.com/news/cmt-patients-specific-rehabilitation-protocol-improves-hand-function/
See for instance 214400
-- Check all of the CMT annotations and write definitions to disambiguate

**Comment by pnrobinson** [comment]
It seems that this term
[Split hand HP:0001171 ](https://hpo.jax.org/app/browse/term/HP:0001171)
has an incorrect synonym of Claw hand. Split hand is ectrodactyly. Some of the annotations seem to be refering to claw (and not split) hand, this will need to be revised carefully.

**Comment by pnrobinson** [comment]
Claw hand deformity
Def: An abormality of the hand characterized by metacarpophalangeal (MCP) hyperextension and proximal interphalangeal (PIP) and distal interphalangeal (DIP) flexion. The position of the affected hand is said to resemble a claw. 
Comment: Anything that may lead to ulnar nerve palsy can cause claw hand. Ulnar nerve palsy can arise from a laceration anywhere along its course. Proximal injuries to the medial cord of the brachial plexus may also present with sensory loss distally. Ulnar nerve palsies can also be due to cubital tunnel syndrome and ulnar tunnel syndrome. These are compression neuropathies at the elbow and wrist. Another cause of ulnar nerve palsy may be a failure to splint the hand in an intrinsic-plus posture following a crush injury. There are a few systemic diseases that may also lead to ulnar nerve palsy. These include leprosy, syringomyelia, and Charcot-Marie-Tooth disease
PMID: 29939558

**Comment by pnrobinson** [comment]
added
